### DISCUSSION TITLE
mount internal drive on server

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#555] [Q&A]
Hello All,

new LTSP server build
256gb NVME drive, 1TB internal HD
Setup the 1TB internal drive to automount at bootup on server. What is the best practice to have this drive show up automagically on the Setup a samba share for the mount point. What do I need to make this drive show up on each booted client at login?

Thank You

**Comment by alkisg** [comment]
You can use /etc/fstab for network shares (either samba, nfs or sshfs); you need to google that part, it's not related to LTSP.
After you find the correct /etc/fstab line, then you can use an FSTAB_="..." line in ltsp.conf to send that fstab line to the clients.

**Comment by brcisna** [comment]
Hi alkisg,

I tried what you suggested.
I have the internal HD mounted on server at /mnt/sda.
in server fstab file this makes the drive shows up and accessible
UUID=7f2ec9bf-a0e0-401a-a6aa-48b81ffe1fee	/mnt/sda	auto nosuid,nodev,nofail,x-gvfs-show 0 0

In /etc/ltsp/ltsp.conf i have this added under [client] section
FSTAB_="UUID=7f2ec9bf-a0e0-401a-a6aa-48b81ffe1fee	/mnt/sda	auto nosuid,nodev,nofail,x-gvfs-show 0 0 "

then ,,  sudo ltsp image /  & sudo ltsp initrd

the internal drive does not show up on client. Am sure i am missing a  step.

Thank You

**Comment by alkisg** [comment]
Put a name there, e.g. FSTAB_MOUNT_SDA=..., don't leave it as plain FSTAB_.
If you run `cat /etc/fstab` on the client, do you see the line there?
If you run `ls /mnt/sda` on the client, does the directory exist?
If you manually run `sudo mount /mnt/sda` on the client, does it get mounted?

**Reply by brcisna** [reply]
brcisna@ltsp110:~$ cat /etc/fstab
# Generated by `ltsp init`, see man:ltsp(8)
UUID=7f2ec9bf-a0e0-401a-a6aa-48b81ffe1fee	/mnt/sda	auto nosuid,nodev,nofail,x-gvfs-show 0 0 
brcisna@ltsp110:~$ ls /mnt/sda
ls: cannot access '/mnt/sda': No such file or directory
brcisna@ltsp110:~$ 

alkisg,

Had set up a samba share to this drive as as 'storage'
have the folder set to 0777 ( until make sure it works) 

i also tried smb://server ip/storage
in the ltsp.conf file

same result

Thanks

**Comment by alkisg** [comment]
So the problem is that you don't have a /mnt/sda folder, you need to create it before you can mount a partition there. The /etc/fstab file doesn't create directories automatically.

**Reply by brcisna** [reply]
alkis ,

yes there is an /mnt/sda  folder  set permissions to 0777 to try and get to work 

thanks

**Comment by alkisg** [comment]
@brcisna I'm not sure if you mean that you have resolved the issue or not; if you didn't, you may come to the chat room for live support:
https://ltsp.org/advanced/chat-room/

The permissions of the mount point do not matter, but your `ls /dev/sda` above said the directory doesn't exist on the client.
